priv/repo/migrations/20220319224657_add_h3_to_profiles.exs

```diff
@@ -0,0 +1,27 @@
+defmodule T.Repo.Migrations.AddhH3ToProfiles do
+  use Ecto.Migration
+
+  import Ecto.Query
+  import Ecto.Changeset
+  alias T.Feeds.FeedProfile
+
+  def change do
+    alter table(:profiles) do
+      add :h3, :decimal
+    end
+
+    flush()
+
+    FeedProfile
```

Comment:
Migrations shouldn't use ecto schemas, since they tend to change, and at some point in the future this migration wouldn't work.

```
"profiles"
|> select([p], {p.user_id, p.location})
|> where([p], not is_nil(p.location))
|> Repo.all()
|> Enum.chunk_by(300)
|> Enum.each(fn chunk ->
  updates = Enum.map(chunk, fn {user_id, {lon, lat}} ->
    [user_id: user_id, h3: :h3.from_geo({lat, lon}, 10)]
  end)
  Repo.insert_all("profiles", updates, on_conflict: {:replace, [:h3]}, conflict_target: :user_id)
end)
```